src/api/db/migrate/001_initial_database.rb

```diff
@@ -897,10 +897,6 @@ def self.up
       INSERT INTO `db_project_types` VALUES (1,'standard'),(2,'maintenance'),(3,'maintenance_incident'),(4,'maintenance_release');
     SQL
 
-    execute <<-SQL
```

Comment:
Is it much work for you to move this to another commit? Because it seems unrelated to the other changes!